docker-compose-cginf.yml

```
@@ -56,10 +56,10 @@
         AIRFLOW__WEBSERVER__SECRET_KEY: '42'
         AIRFLOW__WEBSERVER__WORKERS: 1
       volumes:
-        # - ../airflow-dags/dags:/opt/airflow/dags
-        # - ../FastETL:/opt/airflow/plugins/FastETL
-        # - ../airflow_commons:/opt/airflow/plugins/airflow_commons
-        # - ../airflow-dags/great_expectations:/opt/airflow/great_expectations
+        - ../airflow-dags/dags:/opt/airflow/dags
+        - ../FastETL:/opt/airflow/plugins/FastETL
+        - ../airflow_commons:/opt/airflow/plugins/airflow_commons
+        - ../airflow-dags/great_expectations:/opt/airflow/great_expectations
         - /var/run/docker.sock:/var/run/docker.sock
         - /usr/bin/docker:/bin/docker:ro
```

```suggestion
        # - /usr/bin/docker:/bin/docker:ro
```

Já que o `docker-compose` é para o ambiente local, já deveria deixar a configuração preparada para este tipo de ambiente.

Sem comentar essas linhas dá um erro no ambiente local, pois o `/usr/bin/docker` é somente leitura.